src/analysis/modals/HailBatchModal.test.ts

```diff
@@ -53,11 +55,12 @@ describe('HailBatchModal', () => {
 
     // Act
     await act(async () => {
-      await render(h(HailBatchModal, defaultHailBatchProps));
-      const createButton = screen.getByText('Create');
-      await userEvent.click(createButton);
+      render(h(HailBatchModal, defaultHailBatchProps));
     });
 
+    const createButton = screen.getByText('Create');
+    await user.click(createButton);
```

Comment:
Removing `act()` from all tests in this file seems to still work. For example, this works for this test:
```
    render(h(HailBatchModal, defaultHailBatchProps));
    const createButton = screen.getByText('Create');
    await userEvent.click(createButton);
```

My understanding is that `render()` and `click()` call `act()` internally so you don't need to do it here as well.

Reply:
Removed these acts in https://github.com/DataBiosphere/terra-ui/pull/4059/commits/80f2dfab272f4e8d62262f6b360e612ba005136d.